elasticsearch5/src/main/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndex.java

```diff
@@ -135,22 +135,13 @@ protected Client createClient(ElasticsearchSearchIndexConfiguration config) {
     private Client createInProcessNode(ElasticsearchSearchIndexConfiguration config) {
         Settings settings = tryReadSettingsFromFile(config);
         if (settings == null) {
-            String dataPath = config.getInProcessNodeDataPath();
-            checkNotNull(dataPath, ElasticsearchSearchIndexConfiguration.IN_PROCESS_NODE_DATA_PATH + " is required for in process Elasticsearch node");
-            String logsPath = config.getInProcessNodeLogsPath();
-            checkNotNull(logsPath, ElasticsearchSearchIndexConfiguration.IN_PROCESS_NODE_LOGS_PATH + " is required for in process Elasticsearch node");
-            String workPath = config.getInProcessNodeWorkPath();
-            checkNotNull(workPath, ElasticsearchSearchIndexConfiguration.IN_PROCESS_NODE_WORK_PATH + " is required for in process Elasticsearch node");
-            int numberOfShards = config.getNumberOfShards();
+            String homePath = config.getInProcessNodeHomePath();
+            checkNotNull(homePath, ElasticsearchSearchIndexConfiguration.IN_PROCESS_NODE_HOME_PATH + " is required for in process Elasticsearch node");
 
             Map<String, String> mapSettings = new HashMap<>();
-            mapSettings.put("script.inline", "true");
-            mapSettings.put("index.number_of_shards", Integer.toString(numberOfShards));
```

Comment:
How do I configure this? I kinda need this.

Reply:
In ES5, index level settings can't be on node level configurations. The properties in the configuration file are still the same. We set the values on the creation of indices: https://github.com/visallo/vertexium/blob/master/elasticsearch5/src/main/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndex.java#L1853-L1856